arrays/arrays.c

```diff
@@ -19,45 +19,51 @@ typedef struct Array {
 /*****
  * Allocate memory for a new array
  *****/
-Array *create_array (int capacity) {
-  // Allocate memory for the Array struct
+Array *create_array(int capacity)
+{
+    // Allocate memory for the Array struct
+    Array *newArr = malloc(sizeof(Array));
 
-  // Set initial values for capacity and count
+    // Set initial values for capacity and count
+    newArr->capacity = capacity;
+    newArr->count = 0;
 
-  // Allocate memory for elements
+    // Allocate memory for elements
+    newArr->elements = malloc(capacity * sizeof(char));
```

Comment:
`sizeof(char *)` :) remember this is an array of `char *`, not an array of chars

Reply:
Outdated commit ;P